definitions/benefit/sheets/CaseEventToComplexTypes/CaseEventToComplexTypes.json

```diff
@@ -1492,6 +1503,159 @@
+  },
+  {
+    "CaseEventID": "addHearing",
+    "CaseFieldID": "hearings",
+    "ListElementCode": "venue.name",
+    "ID": "addHearingVenueName",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "FieldDisplayOrder": 2,
+    "PageID": "1.0"
+  },
+  {
+    "CaseEventID": "addHearing",
+    "CaseFieldID": "hearings",
+    "ListElementCode": "venue.address",
+    "ID": "addHearingVenueAddress",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "FieldDisplayOrder": 3,
+    "PageID": "1.0"
+  },
+  {
+    "CaseEventID": "addHearing",
+    "CaseFieldID": "hearings",
+    "ListElementCode": "venue.address.line1",
+    "ID": "addHearingVenueAddressLine1",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "FieldDisplayOrder": 4,
+    "PageID": "1.0"
+  },
+  {
+  "CaseEventID": "addHearing",
+  "CaseFieldID": "hearings",
+  "ListElementCode": "venue.address.line2",
+  "ID": "addHearingVenueAddressLine2",
+  "DisplayContext": "MANDATORY",
+  "LiveFrom": "01/01/2018",
+  "FieldDisplayOrder": 5,
+  "PageID": "1.0"
+},
+{
+  "CaseEventID": "addHearing",
+  "CaseFieldID": "hearings",
+  "ListElementCode": "venue.address.line3",
+  "ID": "addHearingVenueAddressLine3",
+  "DisplayContext": "MANDATORY",
+  "LiveFrom": "01/01/2018",
+  "FieldDisplayOrder": 6,
+  "PageID": "1.0"
+  },
+  {
+    "CaseEventID": "addHearing",
+    "CaseFieldID": "hearings",
+    "ListElementCode": "venue.address.town",
+    "ID": "addHearingVenueAddressTown",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "FieldDisplayOrder": 7,
+    "PageID": "1.0"
+  },
+  {
+    "CaseEventID": "addHearing",
+    "CaseFieldID": "hearings",
+    "ListElementCode": "venue.address.county",
+    "ID": "addHearingVenueAddressCounty",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "FieldDisplayOrder": 8,
+    "PageID": "1.0"
+  },
+  {
+    "CaseEventID": "addHearing",
+    "CaseFieldID": "hearings",
+    "ListElementCode": "venue.address.postcode",
+    "ID": "addHearingVenueAddressPostcode",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "FieldDisplayOrder": 9,
+    "PageID": "1.0"
+  },
+  {
+    "CaseEventID": "addHearing",
+    "CaseFieldID": "hearings",
+    "ListElementCode": "venue.address.country",
+    "ID": "addHearingVenueAddressCountry",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "FieldDisplayOrder": 10,
+    "PageID": "1.0"
+  },
+  {
+    "CaseEventID": "addHearing",
+    "CaseFieldID": "hearings",
+    "ListElementCode": "venue.address.inMainlandUk",
+    "ID": "addHearingVenueAddressLivingInTheUk",
+    "DisplayContext": "OPTIONAL",
+    "FieldShowCondition": "appeal.benefitType.code=\"infectedBloodAppeal\"",
+    "LiveFrom": "01/01/2018",
+    "FieldDisplayOrder": 11,
+    "PageID": "1.0"
+  },
+  {
+    "CaseEventID": "addHearing",
+    "CaseFieldID": "hearings",
+    "ListElementCode": "venue.address.portOfEntry",
```

Comment:
this one shouldn't be here as we populate it in the backend. please remove this field from the event